packages/app/src/layouts/default.vue

```diff
@@ -15,7 +15,13 @@
       :page-name="currentRoute.name?.toString()"
       data-cy="app-header-bar"
       :allow-automatic-prompt-open="true"
-    />
+    >
+      <template #banner>
```

Comment:
I wonder whether it would be better to put the `v-if` on the `template` in this case, that way `HeaderBar` could style itself differently based on whether `banner` slot content was provided or not. As-is there's always going to be *something* provided to that slot, even if it's empty content.

Nbd right now, more food for thought